recipes/msfragger/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# These values are submitted to the MSFragger site when downloading the application zip.
+if [[ -z "$NAME" ]]; then
+    NAME="${USERNAME:-bioconda}";
+fi
+if [[ -z "$EMAIL" ]]; then
+    EMAIL="${NAME}@${HOSTNAME:-bioconda.org}";
+fi
+if [[ -z "$INSTITUTION" ]]; then
+    INSTITUTION="${HOSTNAME:-bioconda.org}";
+fi
+
+# Create directories
+mkdir -p "$PREFIX/tools"
```

Comment:
Most Bioconda recipes uses:
```bash
TARGET=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
```
for this kind of files.

Reply:
Ah, thanks. I wasn't aware of that convention.